dicom_standard/extract_ciod_module_tables.py

```diff
@@ -22,19 +22,24 @@
     table_to_dict,
 )
 
-CHAPTER_ID = 'chapter_A'
+CHAPTER_IDS = ['chapter_A', 'chapter_F']
 # Standard workaround: Include upper case "S" to catch typo in Table A.39.19-1
 # http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.35.19.3.html
 TABLE_SUFFIX = re.compile(".*IOD Module[sS]$")
-COLUMN_TITLES = ['informationEntity', 'module', 'reference_fragment', 'usage']
+COLUMN_TITLES_WITH_IE = ['informationEntity', 'module', 'referenceFragment', 'usage']
+COLUMN_TITLES_NO_IE = ['module', 'referenceFragment', 'usage', 'description']
 
 
 def is_valid_ciod_table(table_div: Tag) -> bool:
     return bool(TABLE_SUFFIX.match(pr.table_name(table_div)))
 
 
-def ciod_table_to_dict(table: TableListType) -> List[TableDictType]:
-    return table_to_dict(table, COLUMN_TITLES)
+def ciod_table_to_dict(table: StringifiedTableListType) -> List[TableDictType]:
+    # Table F.3-1 (the only table in section F) has no "Information Entity" column, so we check for the href in the second column
+    # http://dicom.nema.org/dicom/2013/output/chtml/part03/sect_F.3.html#table_F.3-1
+    sect_f_table = 'href' in table[0][1]
+    column_titles = COLUMN_TITLES_NO_IE if sect_f_table else COLUMN_TITLES_WITH_IE
+    return table_to_dict(table, column_titles)
```

Comment:
Nice, this workaround is well documented w/ the comment and good variable names + reference to the original issue :+1: